### ISSUE TITLE
Render error when string in example contains '</'

### ISSUE THREAD

**Issue by aleung** [asyncapi/html-template#168]
#### Describe the bug
The content of string in example can not be correctly render when the string contains `</`.

#### How to Reproduce

generator version: 1.6.2
html-template version: 0.18.1

Screenshots
![Selection_018](https://user-images.githubusercontent.com/271538/112462901-93dbc400-8d9c-11eb-92c2-0f006a62c723.png)

Files that can help reproduce the issue

``` yaml
asyncapi: 2.0.0
info:
  title: Test
  version: 1.0.0
channels:
  testcase:
    subscribe:
      message:
        $ref: '#/components/messages/LwM2mOjbects'
components:
  messages:
    LwM2mOjbects:
      payload:
        type: object
        properties:
          objectLinks:
            type: string
        example:
          objectLinks: "lwm2m=1.1, </0/0>, </1/1>;ssid=1, </2>, </3/0>"

```

#### Expected behavior
In generated HTML, the `objectLinks` in example should be shown as same as the value in definition file, containing `</0/0>`...

**Comment by magicmatatjahu** [comment]
@aleung Hi! Thanks for report issue! Most likely it is due to sanitizing example -> here https://github.com/asyncapi/html-template/blob/master/partials/example.html#L20 @derberg Do you know why we sanitize examples?

**Comment by derberg** [comment]
tbh you would have to try to remove the `safe` filter and see what happens with HTML input, maybe it was breaking the UI? 🤷🏼 it was there in the initial version which means for a very long time and I bet that even @fmvilas do not remember why he added it there

**Comment by magicmatatjahu** [comment]
@aleung Hi! Sorry for late response. Currently we are unifying HTML-Template with React component in this PR https://github.com/asyncapi/html-template/pull/166 I fixed this bug in unification. Could you wait for it (max next 2 weeks)?

**Comment by aleung** [comment]
No problem. Thanks! @magicmatatjahu

**Comment by magicmatatjahu** [comment]
Fixed in `@asyncapi/html-template@0.21.0` @aleung Reopen if will you found bugs, and sorry for delay 😅
